Subject: Fix deprecated declarations and unused variables causing FTFBS with -Werror
Author: James Lu <bitflip3@gmail.com>
Forwarded: no
Last-Update: 2017-02-04

Index: osdlyrics/src/tests/ol_app_info_test.c
===================================================================
--- osdlyrics.orig/src/tests/ol_app_info_test.c	2017-02-04 23:54:45.000000000 -0800
+++ osdlyrics/src/tests/ol_app_info_test.c	2017-02-04 23:54:51.000000000 -0800
@@ -189,7 +189,6 @@
 int
 main (int argc, char **argv)
 {
-  g_type_init ();
   init ();
   basic_test ();
   desktop_test ();
Index: osdlyrics/src/tests/ol_lyric_source_test.c
===================================================================
--- osdlyrics.orig/src/tests/ol_lyric_source_test.c	2017-02-04 23:54:45.000000000 -0800
+++ osdlyrics/src/tests/ol_lyric_source_test.c	2017-02-04 23:54:51.000000000 -0800
@@ -263,7 +263,6 @@
 int
 main (int argc, char **argv)
 {
-  g_type_init ();
   source = ol_lyric_source_new ();
   test_list_sources ();
   test_search ();
Index: osdlyrics/src/tests/ol_utils_dbus_test.c
===================================================================
--- osdlyrics.orig/src/tests/ol_utils_dbus_test.c	2017-02-04 23:54:45.000000000 -0800
+++ osdlyrics/src/tests/ol_utils_dbus_test.c	2017-02-04 23:54:51.000000000 -0800
@@ -38,7 +38,6 @@
 int
 main ()
 {
-  g_type_init ();
   test_list_name ();
   test_prop ();
   return 0;
Index: osdlyrics/src/tests/ol_utils_test.c
===================================================================
--- osdlyrics.orig/src/tests/ol_utils_test.c	2017-02-04 23:54:45.000000000 -0800
+++ osdlyrics/src/tests/ol_utils_test.c	2017-02-04 23:54:51.000000000 -0800
@@ -83,7 +83,6 @@
 int
 main (int argc, char **argv)
 {
-  g_type_init ();
   ol_log_set_file ("-");
   test_hashtable ();
   test_traverse ();
Index: osdlyrics/src/ol_osd_window.c
===================================================================
--- osdlyrics.orig/src/ol_osd_window.c	2017-02-04 23:22:47.676943417 -0800
+++ osdlyrics/src/ol_osd_window.c	2017-02-04 23:56:56.778432547 -0800
@@ -45,7 +45,6 @@
 };
 static const int MOUSE_TIMER_INTERVAL = 100;
 
-static const int DEFAULT_LINE_HEIGHT = 45;
 static const int DEFAULT_HEIGHT = 100;
 static const double LINE_PADDING = 0.0;
 static const int BORDER_WIDTH = 5;
